webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
 			{
 				test: /\.js/
 				, exclude: /node_modules/
-				, loader: 'babel'
+				, loader: 'babel-loader'
```

Comment:
[According to this](https://github.com/babel/babel-loader#the-node-api-for-babel-has-been-moved-to-babel-core) use of `babel` as as loader has been deprecated as of webpack 2.x. Nice catch!